### PR TITLE
add zh-CN lang and fix  loginFormEntry layout

### PR DESCRIPTION
**Pull request by malaohu**

### CODE REVIEW

**Comment by julianlam** [comment]
Can you split the `loginFormEntry` changeset into a separate PR? I have a feeling @akhoury used those classes because the CAPTCHA is rather large, so I want his input on it.

File: languages/zh-CN/persona.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "mobile-menu-side": "Switch which side each mobile menu is on",
```
Comment (pitaj) [inline languages/zh-CN/persona.json:2]: Any reason mobile-menu-side isn't translated?

Comment (malaohu) [reply languages/zh-CN/persona.json:2]: I will resubmit a PR to translate this. tks!

**Comment by malaohu** [comment]
> Can you split the `loginFormEntry` changeset into a separate PR? I have a feeling @akhoury used those classes because the CAPTCHA is rather large, so I want his input on it.

my login form show this .
img

**Pull request closed without merging**